client/modules/core/helpers/layout.js

```diff
@@ -19,6 +19,7 @@ Template.registerHelper("reactionTemplate", function (options) {
   const shopId = options.hash.shopId || Reaction.getShopId();
   // get shop info, defaults to current
   const Shop = Collections.Shops.findOne(shopId);
+  const defaultRoles = Collections.Groups.findOne({ slug: "customer", shopId }).permissions;
```

Comment:
Do we need to ensure that defaultRoles are loaded at this point? (e.g. `subscription.ready()`)?